reporter/src/main/kotlin/reporters/EvaluatedModelReporter.kt

```diff
@@ -27,6 +27,8 @@ import org.ossreviewtoolkit.utils.log
 import java.io.OutputStream
 import java.io.Writer
 
+private fun EvaluatedModel.toJson(writer: Writer) = toJson(writer, prettyPrint = true)
```

Comment:
Why can't we use a default argument in the called function instead of this? If there's a reason, please add it to the commit message.

Reply:
If have added a default value because it makes sense for the API and explained why the extension function is required.